packages/jest-console/src/test/index.test.js

```diff
@@ -79,7 +80,9 @@ describe( 'jest-console', () => {
 					console[ methodName ].assertionsNumber
 				).toBeGreaterThanOrEqual( 0 );
 			} );
+			/* eslint-enable jest/no-standalone-expect */
 
+			/* eslint-disable-next-line jest/expect-expect */
 			it( 'captures logging in lifecycle', () => {} );
```

Comment:
What even is this testing? 🤔 Not technically in scope, but considering you might have some insight here, a code comment would be helpful (if it's not encompassed in the comment of `beforeAll`).

Reply:
Everything is explained in `beforeAll` and I added a reference as disable reason.